### DISCUSSION TITLE
Reporting issues with Bottles and Formula running on Apple M1 under Rosetta 2

### DISCUSSION THREAD

**Discussion by sunilshah** [Homebrew/discussions#372] [Everyday usage]
Where should I be reporting issues I encounter in running packages under Rosetta2 on Apple  M1?  

For example,  

 * running Python3 itself under Rosetta2 (` arch -x86_64 python3`)
 * Python3 packages installed using pip3 (`arch -x86_64 pip3 install scikit-learn`) 
 
Secondly, can I start running both arm native and Rosetta2 versions of brew packages on the same machine safely?  
If yes, where are the most current instructions to do so?

Thanks in advance.

**Comment by carlocab** [comment]
There's already a related issue here that you may wish to look at: https://github.com/Homebrew/homebrew-core/issues/67713
